common/app/views/fragments/containers/facia_cards/standardContainer.scala.html

```diff
@@ -14,7 +14,7 @@
         ("js-container--fc-show-more", containerDefinition.addShowMoreClasses),
         ("fc-show-more--mobile-only", containerDefinition.hasMobileOnlyShowMore)
     ))"
-         data-title="@Html(Localisation(containerDefinition.displayName getOrElse ""))"
+         data-title="@Localisation(containerDefinition.displayName getOrElse "")"
```

Comment:
depends what this is used for - might want to keep this as is. pretty sure there's no problem with one in an attribute